eng/pipelines/templates/stages/source-build-stages.yml

```diff
@@ -10,6 +10,8 @@ parameters:
 - name: scope
   type: string
   values:
+  # run only windows jobs
```

Comment:
This doesn't seem necessary - this is for source-build, no windows jobs are applicable.